tests/ga/test_extension.py

```diff
@@ -1961,5 +2026,174 @@ def test_ext_handler_with_systemd_cgroup_enabled(self, *args):
         self._assert_no_handler_status(protocol.report_vm_status)
 
 
+@patch('azurelinuxagent.ga.exthandlers.ExtHandlerInstance._capture_process_output', side_effect="Process Successful")
```

Comment:
Same comment, logic will change

Reply:
This patch was just to avoid waiting for the output. With your change it would be even simpler as I can just remove this patch or add a patch to start_extension_command instead